completions/bash/skopeo

```diff
@@ -183,15 +185,12 @@ _cli_bash_autocomplete() {
 
      _get_comp_words_by_ref -n : cur prev words cword
 
-     local command=${PROG} cpos=0
+     local command="skopeo" cpos=0
      local counter=1
-     counter=1
-     while [ $counter -lt $cword ]; do
+     while [ $counter -lt "$cword" ]; do
          case "${words[$counter]}" in
-             -*)
-                ;;
-             *)
-                 command=$(echo "${words[$counter]}" | sed 's/-/_/g')
+             skopeo|copy|inspect|delete|manifest-digest|standalone-sign|standalone-verify|help|h)
```

Comment:
Hum… this is not really correct, consider `skopeo --policy copy inspect …` and the like. I appreciate that in practice, it’s clearly better than the current code for both filenames and parameter values, but it’s likely to confuse future readers into thinking that this code is correct; maybe worth at least a comment or filing an issue.

Also, this is yet another place to maintain this list… could that do with some of the “glob magic” and based on a helper that calls `--generate-bash-completion`?

(Really the trick of `_skopeo_skopeo` is, in hindsight, a mistake.)

Reply:
It's really nasty to get it correct since the entire logic is split. I don't want to rewrite the script but merely wanted to address the big shellcheck complaints and address the completion bug after --flags.